core/lib/generators/spree/install/install_generator.rb

```diff
@@ -123,8 +123,10 @@ def include_seed_data
     def install_migrations
       say_status :copying, 'migrations'
       silence_stream(STDOUT) do
-        silence_warnings { rake 'railties:install:migrations' }
-        silence_warnings { rake 'acts_as_taggable_on_engine:install:migrations' }
+        silence_warnings { rake 'active_storage:install:migrations' }
```

Comment:
The `railties:install:migrations` task will also install `acts_as_taggable_on` migrations, so we need to run each installation manually instead.